cl/corpus_importer/utils.py

```diff
@@ -99,18 +99,23 @@ def match_lists(
     for i, row in enumerate(scores):
         j = row.argmax()  # type: ignore
         # Lower threshold for small opinions.
-        if (
-            get_cosine_similarity(
-                harvard_opinions_list[i], cl_opinions_list[j]
-            )
-            < 0.60
-        ):
-            continue
+        cosine_sim = get_cosine_similarity(
+            harvard_opinions_list[i], cl_opinions_list[j]
+        )
         percent_match = compare_documents(
             harvard_opinions_list[i], cl_opinions_list[j]
         )
-        if percent_match < 60:
+
+        # Sometimes cosine similarity fails when there are small variations in text,
+        # such as parties, attorneys, case name, or court that are included in the
+        # content of the opinion, compare_documents() checks the percentage of the
+        # harvard opinion text that it is in courtlistener opinion, having a large
+        # percentage means that almost all the harvard opinion is in courtlistener
+        # opinion, but there is a possibility that the courtlistener opinion contains
+        # some additional data in que opinion content (such as case name, parties, etc.)
+        if cosine_sim < 0.60 and percent_match < 60:
```

Comment:
Sometimes we have extra data in the opinion content, for example: https://www.courtlistener.com/opinion/1302226/go/
There we have the citation, the parties, the court, the filed date, attorneys and the opinion itself

But if you take a look to the xml file: https://ia803108.us.archive.org/12/items/law.free.cap.nc.304/585.8569666.json

The only content of the opinion is:
`Per Curiam.\nWe hereby adopt the reasoning of the Court of Appeal’s opinion and affirm its decision in all respects.\nAffirmed.\n`

if you use both algorithms, the cosine similarity will return: 0.535 and compare documents will return: 81%, both results are correct, but in the original code, if the cosine similarity is less than 0.60 we continued with next opinion, but in this case the opinion if the harvard file is a subset of the courtlistener opinion, i thought it shouldn't be ignored, that's why i combined both conditions, to make sure that if we are going to continue with the next opinion, it is because neither of the two algorithms could find a match large enough to think that they are the same opinion